pkg/pillar/kubeapi/storageutils.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package kubeapi
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/lf-edge/eve/pkg/pillar/pubsub"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"golang.org/x/sys/unix"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	longhornDevPath = "/dev/longhorn"
+)
+
+func getMajorMinorStr(stat syscall.Stat_t) string {
+	major := unix.Major(stat.Rdev)
+	minor := unix.Minor(stat.Rdev)
+	return fmt.Sprintf("%d:%d", major, minor)
+}
+
+func isLonghornVolAttachedToThisNode(pv string) bool {
+	if _, err := os.Stat(longhornDevPath + "/" + pv); err != nil {
+		return false
+	}
+	return true
```

Comment:
minor nagging:

```suggestion
	_, err := os.Stat(filepath.Join(longhornDevPath,  pv))
	return err == nil
```
(or use `FileExists` from `utils/file/file.go`)

Reply:
sure, more concise